recipes-samples/packagegroups/packagegroup-rpb-tests.bb

```diff
@@ -38,4 +38,5 @@ RDEPENDS_packagegroup-rpb-tests-console = "\
     stress-ng \
     ptest-runner \
     usbutils \
+    net-snmp \
```

Comment:
we (try to) sort this alphabetically, i am now noticing that ptest-runner is not right.. but can you please add net-snmp correctly?

Reply:
Done.
Thanks, Nicolas.